scripts/monitoring/cron-event-watcher.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+  Process OpenShift event stream
+'''
+#
+#   Copyright 2016 Red Hat Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#This is not a module, but pylint thinks it is.  This is a command.
+#pylint: disable=invalid-name
+# pylint flaggs import errors, as the bot doesn't know out openshift-tools libs
+#pylint: disable=import-error
+
+import argparse
+from openshift_tools.monitoring.zagg_sender import ZaggSender
+import json
+from Queue import Queue
+import re
+import subprocess
+import threading
+import time
+
+EVENT_PERIOD = 60 # seconds
```

Comment:
Any reason not to put this into the config file?  Seems like we could potentially want to change the interval without having to rebuild.